cpp/tests/lists/combine/concatenate_rows_tests.cpp

```diff
@@ -814,17 +814,16 @@ TEST_F(ListConcatenateRowsNestedTypesTest, StructWithNulls)
 
     // expected
     cudf::test::fixed_width_column_wrapper<int> se_0{
-      {10, 11, 12, 13, 2, 3, 4, 14, 15, 16, 5, 17, 18, 19, 20, 21, 22, 23, 24, 6, 7},
-      nulls_at({18})};
+      {10, 11, 12, 13, 2, 3, 4, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 6, 7}, nulls_at({17})};
     cudf::test::strings_column_wrapper se_1{
-      {"arg",    "mno",     "ampere", "gpu",   "bananas", "",      "",    "",    "hhh", "warp", "g",
+      {"arg",    "mno",     "ampere", "gpu",   "bananas", "",      "",    "",    "hhh", "warp",
```

Comment:
Why was this change made?

Reply:
At line 762 above, the `"g"` string corresponds to a null list (line 769, the last null). Before this PR, that null list still store `"g"` which will be concatenated later to the final output. With this PR, all null lists are empty thus `"g"` is removed completely.